isce2_topsapp/templates/topsapp_template.xml

```diff
@@ -17,7 +17,7 @@
     <property name="range looks">{{ range_looks }}</property>
     <property name="azimuth looks">{{ azimuth_looks }}</property>
     <property name="filter strength">{{ filter_strength }}</property>
-    <!-- <property name="region of interest">{{ region_of_interest }}</property> -->
+    <property name="region of interest">{{ region_of_interest }}</property>
```

Comment:
may need to do some jinja logic here as it was commented out from GUNW SLC generation. I think you can just:

```
{% if region_of_interest }<property name="region of interest">{{ region_of_interest }}</property>{% endif %}
```
though you may need to adjust the whitespace control: https://jinja.palletsprojects.com/en/3.1.x/templates/#whitespace-control

Reply:
Hmm, looks like they do specify it for GUNW SLCs and it might just work. @cmarshak why's region of interest commented out when you do compute it and pass it to the template? Looks like it was added that way initially so I don't have any insight there.

Reply:
It's from the original in the ariamh repo and I had messed around with it to reduce processing time for tests - but was annoyed that if it's included then, it cannot be left blank. Jinja would solve it

Reply:
Honestly, I think it's fine like this: https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/9fd15a2beda86b347dd4f4c236cfd38a87b3c2e6/isce2_topsapp/topsapp_proc.py#L42-L57